File: sol/solution/0001-0100/0026/Solution.ts
export function removeDuplicates(nums: number[]): number {
	if (nums.length === 0) return 0;
	let k = 0;
	for (let i = 1; i < nums.length; i++) {
		if (nums[i] !== nums[k]) {
			k++;
			nums[k] = nums[i];
		}
	}
	return k + 1;
}

File: sol/solution/0001-0100/0026/Solution.swift
class Solution {
    func removeDuplicates(_ nums: inout [Int]) -> Int {
        guard nums.count > 1 else { return nums.count }
        var write = 1
        for i in 1..<nums.count {
            if nums[i] != nums[i - 1] {
                nums[write] = nums[i]
                write += 1
            }
        }
        return write
    }
}